### DISCUSSION TITLE
FF 6.1.22 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by pupilov** [firefly-iii/firefly-iii#9457] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hello. After upgrading from 6.1.21 to the latest 6.1.22, Firefly keeps crashing with 500 Internal Server Error :(.
Going back to 6.1.21 fixes the problem, and Firefly starts working again.
I have attached the full log file, it seems it has something to do with the new date/time entries in the DB. 

I will stay on 6.1.21 for now, as I can't afford losing current DB. But maybe my logs can help solve a developing problem.

### Debug information

Debug information generated at 2024-11-11 01:26:00 for Firefly III version **v6.1.21**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.21 / v2.1.0 / 24 (exp. 24)</td></tr><tr><td>PHP version</td><td>8.3.12 (64bits) / apache2handler / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, pgsql, sqlite, </td></tr><tr><td>Docker build</td><td><span>#</span>1131, base <span>#</span>91</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Bucharest + Europe/Bucharest</td></tr><tr><td>App environment</td><td>production, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>3 of 1</td></tr><tr><td>User flags</td><td></td></tr><tr><td>Session start</td><td>2024-11-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-11-30 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36</td></tr></tbody></table>

### Steps to reproduce

1. update to 6.1.22 from 6.1.21
2. check for db errors

### Additional info
[_firefly_iii_core_logs.txt](https://github.com/user-attachments/files/17693504/_firefly_iii_core_logs.txt)

**Comment by JC5** [comment]
Hi, thanks for opening an issue!

The code to upgrade the database is part of the docker container. Please remove the Firefly III container and image completely, and start it again. That should solve the issue.

**Comment by H3wastooshort** [comment]
i had the same issue which got resolved by re-running the migrations completely: https://github.com/orgs/firefly-iii/discussions/9460#discussioncomment-11280163
